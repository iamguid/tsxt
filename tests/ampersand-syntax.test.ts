import template from "../compiled-templates/ampersand-syntax.template";

it("template generates correctly", () => {
  const result = template();
  expect(result).toMatchSnapshot();
});
