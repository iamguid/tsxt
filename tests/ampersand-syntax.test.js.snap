// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template generates correctly 1`] = `
"a
b
d
"
`;
